test/unittest/dmlc_unittest.mk

```diff
@@ -1,13 +0,0 @@
-UTEST_ROOT=test/unittest
-UNITTEST=$(UTEST_ROOT)/dmlc_unittest
-UNITTEST_SRC=$(wildcard $(UTEST_ROOT)/*.cc)
-UNITTEST_OBJ=$(patsubst %.cc,%.o,$(UNITTEST_SRC))
-
-GTEST_LIB=$(GTEST_PATH)/lib/
-GTEST_INC=$(GTEST_PATH)/include/
-
-$(UTEST_ROOT)/%.o : $(UTEST_ROOT)/%.cc libdmlc.a
-	$(CXX) $(CFLAGS) -I$(GTEST_INC) -o $@ -c $<
```

Comment:
I deleted `dmlc_unittest.mk`, since it would attempt to build `unittest_parquet_parser.cc` using GNU Makefile. It is just too much hassle to use GNU Make to build a C++ program with Arrow and Parquet.

Reply:
Acknowledged. Thank you!